README.md

```diff
@@ -96,6 +98,10 @@ If existing code gets reformatted, this must be done in a separate commit. Its c
 ### Running CI tests locally
 To avoid unneeded load on the GitHub infrastructure, please consider running `tools/ci/run_ci.sh --all` before pushing.
 
+### Running integration tests locally
+`cd wakaama`
+`tools/ci/run_ci.sh --run-build`
+`pytest -v`
```

Comment:
Please add a newline here

Reply:
ok

Reply:
will also modify to `pytest -v tests/integration` to remove "PytestUnknownMarkWarning"